### PR TITLE
Cloudflare.cloudflared version 2022.3.1

### PR DESCRIPTION
**Pull request by JohannVR**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/54458)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/c/cloudflare/cloudflared/2022.3.1/cloudflare.cloudflared.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: cloudflare.cloudflared
```
Comment (ItzLevvie) [inline manifests/c/cloudflare/cloudflared/2022.3.1/cloudflare.cloudflared.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Cloudflare.cloudflared
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will also have to update the publisher folder name from `cloudflare` to `Cloudflare`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/54458#pullrequestreview-910309957

You can press the `.` (dot) on your keyboard on this PR to make the changes using Visual Studio Online.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/c/Cloudflare/cloudflared/2022.3.1/Cloudflare.cloudflared.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Cloudflare.cloudflared
+PackageVersion: 2022.3.1
+PackageLocale: en-US
+Publisher: Cloudflare
```
[the remark this answers is not in the document]
Comment (JohannVR) [reply manifests/c/Cloudflare/cloudflared/2022.3.1/Cloudflare.cloudflared.locale.en-US.yaml]: awesome, thanks :)

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**